src/dawn/engine_dawn.h

```diff
@@ -69,6 +75,42 @@ class EngineDawn : public Engine {
     return pipeline_map_[command->GetPipeline()].render_pipeline.get();
   }
 
+  // Helper stuff
+  struct Helper {
```

Comment:
If this is only used internally, move this struct definition to the anonymous namespace of engine_dawn.cpp